client-js/app/elements/labrad-plot.ts

```diff
@@ -562,11 +578,54 @@ export class Plot extends polymer.Base {
               .attr('y', (d) => p.yScale(p.yNext[d[1]]))
               .attr('width', (d) => p.xScale(p.xNext[d[0]]) - p.xScale(d[0]))
               .attr('height', (d) => Math.abs(p.yScale(p.yNext[d[1]]) - p.yScale(d[1])))
-              .style('fill', (d) => getColor(d[2], zMin, zMax));
+              .style('fill', (d) => getColor(d[p.displayTraces[0] + 2], zMin, zMax));
       }
       break;
     }
   }
+  
+  private selectTraces() {
+    this.$.traceSelector.open();
+  }
+
+  private selectAll() {
+    var checkboxes: any = document.getElementsByName('traces');
```

Comment:
It requires a bit of casting since the return type is a collection of generic nodes, but this seems to work:
```typescript
var checkboxes = Polymer.dom(this.$.traceSelector).querySelectorAll('[name=traces]');
for (let checkbox of checkboxes) {
  (<HTMLInputElement>checkbox).checked = true;
}
```